### ISSUE TITLE
FXIOS-1371 ⁃ SPIKE: Click to Call Not Working on Some Sites

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#7814]
Problem:
Click to call feature not working on some sites

Steps to Reproduce:
1. Go to a website with phone numbers listed. For example, https://www.solanocounty.com/depts/genserv/nta/default.asp

Expected Behaviour:
Phone numbers can be clicked to call, just like on Safari 

Actual Behaviour:
Phone numbers cannot be clicked

Notes:

* iOS 14.3 Firefox v30 (unclear when started)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1371)

**Comment by data-sync-user** [comment]
➤ Ashley Thomas commented:

Marking as P1 because this is a high frequency issue

**Comment by dnarcese** [comment]
This worked for me on the Bell site I was testing. Wonder if it is specific to that site?

![IMG_B63D1A86B326-1](https://user-images.githubusercontent.com/11432165/103919728-61e08d00-50de-11eb-8d51-9c80f3e01cf6.jpeg)

**Comment by SimonBasca** [comment]
I've manage to find a reliable link to test this issue : https://shaun.net/resources/test-phone-numbers/

The phone numbers listed work fine in Safari but not in other browsers like Google Chrome or Firefox 30.0 (3297).

**Comment by boek** [comment]
looks like we may be able to fix this with

https://developer.apple.com/documentation/webkit/wkwebviewconfiguration/1641937-datadetectortypes
```
let config = WKWebViewConfiguration()
config.dataDetectorTypes = [.phoneNumber]
```

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verified fix on v32.0 (3815).
